frontend/src/studio-ui/TinyMceEditor.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Editor } from '@tinymce/tinymce-react';
+
+// TinyMCE so the global var exists
+import 'tinymce/tinymce';
+// DOM model
+import 'tinymce/models/dom/model';
+// Theme
+import 'tinymce/themes/silver';
+// Toolbar icons
+import 'tinymce/icons/default';
+// Editor styles
+import 'tinymce/skins/ui/oxide/skin';
+
+// importing the plugin js.
+// if you use a plugin that is not listed here the editor will fail to load
+import 'tinymce/plugins/advlist';
+// import 'tinymce/plugins/anchor';
+import 'tinymce/plugins/autolink';
+// import 'tinymce/plugins/autoresize';
+// import 'tinymce/plugins/autosave';
+// import 'tinymce/plugins/charmap';
+import 'tinymce/plugins/code';
+// import 'tinymce/plugins/codesample';
+import 'tinymce/plugins/directionality';
+// import 'tinymce/plugins/emoticons';
+import 'tinymce/plugins/fullscreen';
+import 'tinymce/plugins/help';
+import 'tinymce/plugins/help/js/i18n/keynav/en';
+import 'tinymce/plugins/image';
+// import 'tinymce/plugins/importcss';
```

Comment:
I'm leaving these commented out so they can be easily and quickly added if needed in the future.